DiscordBot.Modules/CommandModules/HelpModule.cs

```diff
@@ -137,52 +128,53 @@ public void AddCommands(ModuleInfo module, EmbedBuilder builder)
 
         public void AddCommand(CommandInfo command, EmbedBuilder builder)
```

Comment:
What made you doing this refactorization?

Reply:
The Usage field is now fixed. You said that in our Discord, it doesn‘t look good.

Reply:
What do you mean?

Reply:
<img width="223" src="https://user-images.githubusercontent.com/62756994/86800889-c8bfb380-c073-11ea-8949-7ef537ccda4e.png">
<img width="223" src="https://user-images.githubusercontent.com/62756994/86800897-c9f0e080-c073-11ea-8861-f995f368c6f8.png">

Reply:
I mean the usage